packages/web-ui/.storybook/preview.js

```diff
@@ -37,26 +36,6 @@ const customerUiViewports = {
   ...INITIAL_VIEWPORTS,
 };
 
-const StoriesContainer = props => {
-  const theme = useTheme();
-  useEffect(() => {
-    if (typeof window !== 'undefined') {
-      console.log(
-        `%c
-    ░█▒█░█░░▒█░░░█░░▒█▒██▀░██▄░░░█▒█░█
-    ░▀▄█░▀▄▀▄▀▒░░▀▄▀▄▀░█▄▄▒█▄█▒░░▀▄█░█
```

Comment:
What is going on here?

Reply:
😆 this was kind of an easter egg I stole off MUI...
<img width="821" alt="image" src="https://github.com/utilitywarehouse/design-systems/assets/9257284/71d04a42-1632-4996-8e9c-9e3dde340766">

It was  a bit flaky, and without the global `ThemeProvider` wouldn't work. Plus as we're moving away from using the theme I figured it was best not to encourage it's use. 🤷🏻

Reply:
Haha oh yeah, love it! Gets your attention